shared/entities/project-key-costs.view.ts

```diff
@@ -0,0 +1,117 @@
+import { ValueTransformer, ViewColumn, ViewEntity } from "typeorm";
+
+export const decimalTransformer: ValueTransformer = {
+  to: (value: number | null) => value,
+  from: (value: string | null): number | null =>
+    value !== null ? parseFloat(value) : null,
+};
+
+@ViewEntity({
+  name: "project_scorecard_key_costs",
+  expression: `
+SELECT
+  p.project_name AS project_name,
+  p.conservation_planning_npv AS conservation_planning_npv,
+  p.conservation_planning AS conservation_planning,
+  p.community_representation_npv AS community_representation_npv,
+  p.community_representation AS community_representation,
+  p.implementation_labor_npv AS implementation_labor_npv,
+  p.implementation_labor AS implementation_labor,
+  p.monitoring_maintenance_mpv AS monitoring_maintenance_mpv,
+  p.monitoring_maintenance AS monitoring_maintenance,
+  p.community_benefit_npv AS community_benefit_npv,
+  p.community_benefit AS community_benefit,
+  p.carbon_standard_fees_npv AS carbon_standard_fees_npv,
+  p.carbon_standard_fees AS carbon_standard_fees,
+  p.long_term_project_operating_npv AS long_term_project_operating_npv,
+  p.long_term_project_operating AS long_term_project_operating
+FROM
+	projects p
+LEFT JOIN
+	project_scorecard ps 
+ON
+	p.country_code = ps.country_code and
+	ps."ecosystem"::VARCHAR = p."ecosystem"::VARCHAR`,
```

Comment:
Additionally, is the join with scorecards really required? the projects entity already has a ecosystem prop which can be brought to the view avoiding the join

Thanks @agnlez for the catch :)